index.js

```diff
@@ -68,7 +68,28 @@ function toggleColor(element) {
   }
 }
 
-/***** Deliverable 2 *****/
+document.querySelector("#header").addEventListener("click", function(e) {
+  toggleColor(e.target)
+})
 
+/***** Deliverable 2 *****/
+const like = document.querySelector('.like-button')
+like.addEventListener('click', function(){
+  const likes = document.querySelector('.likes')
+  const newlike = parseInt(likes.textContent) + 1
+  likes.textContent = newlike + ' likes'
+})
 
 /***** Deliverable 3 *****/
+
+const newGoal = document.querySelector('#new-goal-form')
+newGoal.addEventListener('submit', function(e){
+  e.preventDefault()
+  const goal = {
+    id: player.goals.id,
```

Comment:
`player.goals.id` will return undefined because `player.goals` is an array of goals. You're on the right track though. You could fix is with something like this:

```javascript
player.goals[player.goals.length - 1].id
```